shared/guide/system/software/library/library.group

```diff
@@ -0,0 +1,7 @@
+documentation_complete: true
+
+title: 'Required Libraries'
+
+description: |-
+    The following sections contain information on required libraries.
+
```

Comment:
Since this is for SAP configurations, let's make this file `sap.group` and change the title and description to talk about SAP. That way, we understand what this group is for.